src/client.rs

```diff
@@ -317,27 +319,37 @@ impl fmt::Debug for RequestBuilder {
 }
 
 /// A Response to a submitted `Request`.
+#[derive(Debug)]
```

Comment:
Hm, what I meant was that the debug output probably doesn't need to leak the details of what kind of decoder it is using.

```rust
println!("res = {:?}", res);
```

Ideally that prints something like:

```
res = Response { status: Ok, headers: { ... }, version: Http11, }
```